internal/events/types/user/user_update_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package user_update
+
+import (
+    "encoding/json"
+    "testing"
+
+    "github.com/twitchdev/twitch-cli/internal/events"
+    "github.com/twitchdev/twitch-cli/internal/models"
+    "github.com/twitchdev/twitch-cli/test_setup"
+)
+
+var toUser = "4567"
+
+func TestEventSub(t *testing.T) {
+    a := test_setup.SetupTestEnv(t)
+
+    params := *&events.MockEventParameters{
+        UserID:     toUser,
```

Comment:
Do you mean either the FromUser or ToUser? That's why the tests are failing atm.

Reply:
I used ToUser in the lib and the test here

see
https://github.com/BarryCarlyon/twitch-cli/blob/user-update/internal/events/types/user/user_update.go#L40
https://github.com/BarryCarlyon/twitch-cli/blob/user-update/internal/events/types/user/user_update.go#L50

The test useds line 14 in here to spool up.

And my layout of the lib and test matches other types in here.

So I'm none the wiser here

Reply:
oh I think I understand

Reply:
nope it's tripped again :-|